src/applications/appeals/995/utils/helpers.js

```diff
@@ -1,7 +1,8 @@
 import moment from 'moment';
 
-import { SELECTED, LEGACY_TYPE, FORMAT_YMD, AMA_DATE } from '../constants';
```

Comment:
`AMA_DATE` is already in the shared constants; but let's not worry about it in this PR. It's got enough going on already!